index.html

```diff
@@ -0,0 +1,165 @@
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>PORTFOLIO MOBILE VERSION</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./main.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+
+<body>
+    <section>
+        <div class="container">
+            <div class="card">
+                <div class="card__header">
+                    <img src="./images/Portfolio.png" alt="card__image" class="card__image" width="600">
+                </div>
+                <div class="card__body">
+                    <h1 class="tonic">Tonic</h1>
+                    <h5><b>CANOPY</b> <span>&#183;</span> Back end Dev <span>&#183;</span> 2015</h5>
+                    <p class="paratext">A daily selection of privately personalised reads no accounts or signups required</p>
+                </div>
+                <div class="card__footer">
+                    <div class="user">
+                        <div class="user__info">
+                            <ul>
+                                <li class="social-media-icons"><button type="button" class="btn">html</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">css</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">javascript</button></li>
+                            </ul>
+                            <h5><button type="button" class="btn-b">See Project</button></h5>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card__header">
+                    <img src="./images/Portfolio2.png" alt="card__image" class="card__image" width="600">
+                </div>
+                <div class="card__body">
+                    <h1 class="tonic">Tonic</h1>
+                    <h5><b>CANOPY</b> <span>&#183;</span> Back end Dev <span>&#183;</span> 2015</h5>
+                    <p class="paratext">A daily selection of privately personalised reads no accounts or signups required</p>
+                </div>
+                <div class="card__footer">
+                    <div class="user">
+                        <div class="user__info">
+                            <ul>
+                                <li class="social-media-icons"><button type="button" class="btn">html</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">css</button></li>
+                                <li class="social-media-icons"><button type="button" class="btn">javascript</button></li>
+                            </ul>
+                            <h5><button type="button" class="btn-b">See Project</button></h5>
```

Comment:
[ Required Changes ]
- [ ] Please let the button stand alone by itself. You shouldn't place a button inside a heading tag. Please fix this

Reply:
Done.